test/jdk/javax/management/MBeanServer/ExceptionTestNulls.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8364227
+ * @summary Test various null parameters and verify Exceptions thrown
+ * @modules java.management.rmi
+ * @run main ExceptionTestNulls
```

Comment:
The test looks okay but I wonder if you've considered making it a JUnit test?

Reply:
Thanks for the CSR update.

I was just following the norm for the area with the test.
_Should_ I make this a junit test, is that required, desirable, unnecessary...

Reply:
Great, yes I'm just fishing for other rules.   Happy to leave this test in the same form as its companion tests at the moment.

Many thanks for quick review here and on the CSR.